tutorials/vue/vue-file-upload.md

```diff
@@ -0,0 +1,9 @@
+Using Vue 2 with Axios, you can upload files easily with a few clicks.
+Using the `<input>` tag and specifying the type to file, the browser will
+allow you to select the file to upload from your computer. You will need
```

Comment:
This implies that you need `form-data` to create an HTML form, which isn't quite true.